docs/StardustDocs/topics/cast.md

```diff
@@ -1,18 +1,18 @@
 [//]: # (title: cast)
 <!---IMPORT org.jetbrains.kotlinx.dataframe.samples.api.Modify-->
 
-Changes type argument of [`DataFrame`](DataFrame.md) without changing its contents.
+Changes the type argument of the [`DataFrame`](DataFrame.md) instance without changing its contents.
 
 ```kotlin
 cast<T>(verify = false)
 ```
 
 **Parameters:**
 * `verify: Boolean = false` —
-  when `true`, throws exception if [`DataFrame`](DataFrame.md) doesn't match the given schema. 
-Otherwise, just change a format type without actual data check.
+  when `true`, the function throws an exception if the [`DataFrame`](DataFrame.md) instance doesn't match the given schema. 
+Otherwise, it just changes the format type without actual data checks.
 
-Use this operation to change a formal type of [`DataFrame`](DataFrame.md)
+Use this operation to change the formal type of a [`DataFrame`](DataFrame.md) instance
```

Comment:
So, better to go through all the docs and add instance to all places where it should be applied

Reply:
let's do that afterward though. Currently, I'm just going through your changes and checking and replacing grammar there. When I'm done, I'll do a ctrl+r on the entire thing :) Either here or a separate PR